jupyterlab_latex/__init__.py

```diff
@@ -233,5 +287,7 @@ def load_jupyter_server_extension(nb_server_app):
         nb_server_app (NotebookWebApplication): handle to the Notebook webserver instance.
     """
     web_app = nb_server_app.web_app
-    host_pattern = '.*$'
-    web_app.add_handlers(host_pattern, [(r'/latex%s' % path_regex, LatexHandler)])
+    base_url = web_app.settings['base_url']
+    endpoint = url_path_join(base_url, 'latex')
+    handlers = [(endpoint + "(.*)", LatexHandler)]
```

Comment:
could be a format string rather than a concatenation (since we're already using them elsewhere)

Reply:
f"{endpoint}{path_regex}"?